ui/src/components/layout/Labels.vue

```diff
@@ -29,6 +29,23 @@
                 default: true
             }
         },
+        // this is needed as flows uses a Map and Execution a List of Labels.
+        // if we align both of them this can be removed
+        mounted() {
+            if (Array.isArray(this.labels)) {
+                const labelMap = {};
+                this.labels.forEach(label => labelMap[label.key] = label.value)
+                this.labelMap = labelMap;
```

Comment:
this.labelMap = Object.fromEntries(this.labels.map(label => [label.key, label.value]))

Reply:
Well, maybe but as I don't want to re-test the UI I'll keep this one as it does the same.

Reply:
Made the change and tried it, works :)